Makefile

```diff
@@ -31,7 +31,7 @@ dev: checklist check explaintest gogenerate br_unit_test test_part_parser_dev ut
 # Install the check tools.
 check-setup:tools/bin/revive
 
-check: parser_yacc check-parallel lint tidy testSuite errdoc check-bazel-prepare
+check: parser_yacc check-parallel lint tidy testSuite errdoc check-bazel-prepare check-testcase-marker
```

Comment:
@wuhuizuo please help check if it is appropriate to add here and whether it can serve as a validation before PR merge?

Reply:
yes, it's valid

Reply:
Used here for the testmarker [config](https://github.com/pingcap/tidb/blob/33fc5944a95a91b79c8652120bbed47e92d28540/tests/testmarker/.testmarker.yaml) : https://github.com/pingcap/tidb/blob/33fc5944a95a91b79c8652120bbed47e92d28540/tests/testmarker/walker.go#L34